src/v2/view-builder/views/ov/SwitchEnrollChannelLinkView.js

```diff
@@ -7,7 +7,7 @@ export default View.extend({
   template: hbs `
     {{#if isQRcodeChannel}}
       <a href="#" class="switch-channel-link">
-        {{{i18n code="oie.enroll.okta_verify.qrcode.cannotScan" bundle="login"}}}</a>
+        {{{i18n code="enroll.totp.cannotScan" bundle="login"}}}</a>
```

Comment:
While we're here, can we remove the triple brackets? This shouldn't be HTML unescaped.